### PR TITLE
limit file size

### PR DESCRIPTION
**Pull request by bgruening**
xref: https://github.com/usegalaxy-eu/issues/issues/374

### CODE REVIEW

**Comment by bgruening** [comment]
ping @sj213

**Comment by sj213** [comment]
"Too many blank lines" by Lint??? I don't get it. You *can* have blank lines in YAML. And I don't see an obvious YAML syntax erroe either...

**Comment by sj213** [comment]
Ah, maybe the value of `docker_run_extra_arguments` must be quoted. '-' is a special character in YAML.

**Comment by sj213** [comment]
Ok, you did it already...

**Comment by sj213** [comment]
Could it be that lint barfs on trailing blank lines? In the history I see that Mira had once removed a trailing blank line with the commit comment "lint". Also, the other YAML files in the dir all end with non-blank lines.

**Comment by bgruening** [comment]
> Could it be that lint barfs on trailing blank lines?

yes, this can be.

**Comment by sj213** [comment]
In fact, I just pushed a commit to remove the trailing blank line and lint passed... XD

**Comment by bgruening** [comment]
Thanks!

File: files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/tools.yml [hunk 1]
```diff
@@ -145,3 +145,49 @@ tools:
       require:
         - docker
         - embedded-pulsar
+
```
Comment (mira-miracoli) [inline files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/tools.yml:148]: mothur_cluster_split kann dann raus, oder?

**Review by mira-miracoli — APPROVED** [review]
Looks good to me

**Pull request merged**